spec_linux.go

```diff
@@ -127,6 +127,13 @@ type CPU struct {
 	Mems string `json:"mems"`
 }
 
+type Pids struct {
+	// Maximum number of PIDs. A value <= 0 implies "no limit".
+	// While technically 0 means "no tasks allowed in the cgroup", this does
+	// not make sense in the context of containers.
```

Comment:
I guess that's a fair point. It does make it slightly annoying because the default value will have to be `-1` (which means we can't do `new(spec.Linux)` without modifying it each time). I'll fix this later.